docs/aca/09-aca-autoscale-keda/index.md

```diff
@@ -3,42 +3,45 @@ canonical_url: https://bitoftech.net/2022/09/22/azure-container-apps-auto-scalin
+### An Overview of KEDA
+KEDA stands for Kubernetes Event-Driven Autoscaler. It is an open-source project initially started by [Microsoft and Red Hat](https://cloudblogs.microsoft.com/opensource/2019/05/06/announcing-keda-kubernetes-event-driven-autoscaling-containers/) to allow any Kubernetes workload to benefit from the event-driven architecture model. Prior to KEDA, horizontally scaling Kubernetes deployment was achieved through the Horizontal Pod Autoscaler ([HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)). The HPA relies on resource metrics such as Memory and CPU to determine when additional replicas should be deployed. In an enterprise application, there may be additional external metrics that we want to use to scale our application, such as the length of a Kafka topic log, an Azure Service Bus Queue, or metrics obtained from a Prometheus query. KEDA offers more than [50 scalers](https://keda.sh/docs/2.8/scalers/) to pick from based on your business need. KEDA exists to fill this gap and provides a framework for scaling based on events in conjunction with HPA scaling based on CPU and Memory.
 
 ### Configure Scaling Rule in Backend Background Processor Project
-We need to configure our Backend Background Processor named `tasksmanager-backend-processor` service to scale out and increase the number of replicas based on the number of messages in the Topic named `tasksavedtopic`. If our service is under a huge workload and our single replica is not able to cope with the number of messages on the topic, we need the Container App to spin up more replicas to parallelize the processing of messages on this topic.
+We need to configure our Backend Background Processor `tasksmanager-backend-processor` service to scale out and increase the number of replicas based on the number of messages in the Topic named `tasksavedtopic`. When our service is under heavy workload and a single replica is insufficient to handle the number of messages on the topic, we require the Container App to create additional replicas to distribute the processing of messages on this topic.
 
 So our requirements for scaling the backend processor are as follows:
 
 * For every 10 messages on the Azure Service Bus Topic, scale-out by one replica.
 * When there are no messages on the topic, scale-in to a one single replica.
 * The maximum number of replicas should not exceed 5.
 
-To achieve this, we will start looking into KEDA Azure Service Bus scaler, This specification describes the `azure-servicebus` trigger for Azure Service Bus Queue or Topic, let's take a look at the below yaml file which contains a template for the KEDA specification:
+To achieve this, we will start looking into KEDA Azure Service Bus scaler. This specification describes the `azure-servicebus` trigger for Azure Service Bus Queue or Topic. Let's take a look at the yaml file below which contains a generic template for the KEDA specification:
 ```yaml
 triggers:
```

Comment:
Since this is a complete yaml, it could make sense to move it as a yaml file in the same directory and refer it here.